src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected LocalDate deadline;
+
+    Deadline(String description, String deadline) {
+        super(description);
+        this.deadline = LocalDate.parse(deadline);
+    }
+
+    Deadline(String description, boolean isDone, String deadline) {
+        super(description, isDone);
+        this.deadline = LocalDate.parse(deadline);
+    }
+
+    private String dateToString() {
```

Comment:
The java coding standard says that "Names representing methods must be verbs and written in camelCase." dateToString() in Deadline.java and Event.java don't fit that in my opinion. Maybe something you'd like to consider and change to be more in line with the standard?